vip-application-importer/src/main/java/fr/insalyon/creatis/vip/applicationimporter/client/view/applicationdisplay/VIPLayout.java

```diff
@@ -161,7 +176,29 @@ private SelectItem createTagsSelect() {
         tagsCb.setTitle("<b>Dirac tag</b>");
         tagsCb.setType("comboBox");
 
-        tagsCb.setValueMap("None", "diracTag:nvidiaGPU");
+        final AsyncCallback<List<String>> callback = new AsyncCallback<List<String>>() {
+            @Override
+            public void onFailure(Throwable caught) {
+                Layout.getInstance().setWarningMessage("Unable to retrieve configurated list of requirements, setting it to None:<br />" + caught.getMessage());
+                tagsCb.setValueMap("None");
+            }
+
+            @Override
+            public void onSuccess(List<String> result) {
+                if(!result.contains("None")){
+                    result.add("None");
+                }
+                
+                LinkedHashMap<String, String> requirementsValues = new LinkedHashMap<String, String>();
```

Comment:
Better to use the Map type for the declaration. 
And the diamond <> on construction should work. 
So I suggest
`Map<String, String> requirementsValues = new LinkedHashMap<>();`

Reply:
Thank you for the suggestion. Done